src/Exceptions/InvalidArgumentCastTypeException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace League\CLImate\Exceptions;
+
+class InvalidArgumentCastTypeException extends \Exception implements Exception
+{
+}
```

Comment:
It looks like you're missing the last line ending from the new files you've added

Reply:
Thanks for pointing. Forgot to use phpcs.